core/src/ast/mod.rs

```diff
@@ -203,6 +206,48 @@ impl ToSql for Statement {
             Statement::AlterTable { name, operation } => {
                 format!("ALTER TABLE {} {}", name.to_sql(), operation.to_sql())
             }
+            Statement::DropTable { if_exists, names } => {
+                let names = names
+                    .iter()
+                    .map(ToSql::to_sql)
+                    .collect::<Vec<_>>()
+                    .join(", ");
+                match if_exists {
+                    true => format!("DROP TABLE IF EXISTS {}", names),
+                    false => format!("DROP TABLE {}", names),
+                }
+            }
+            #[cfg(feature = "index")]
+            Statement::CreateIndex {
+                name,
+                table_name,
+                column: _column,
```

Comment:
[nit] If not used, the field can be omitted.
```suggestion
                ..
```

Reply:
In this case, it would be good to specify `column` explicitly for reminder which we will handle this field 'column' soon